workshops/views.py

```diff
@@ -388,7 +391,10 @@ class PersonCreate(LoginRequiredMixin, CreateViewContext):
 
 class PersonUpdate(LoginRequiredMixin, UpdateViewContext):
     model = Person
-    fields = PERSON_FIELDS
+    # don't display the 'password' field, reorder fields
+    fields = ["personal", "middle", "family", "username", "may_contact",
+              "email", "gender", "airport", "github", "twitter", "url",
+              "is_superuser", "user_permissions"]
```

Comment:
PEP 8 doesn't suggest single-vs.-double quotes, but[ does suggest consistency][1].  I personally prefer single quotes, since they have fewer distracting pixels, but other folks [seem][2] to [prefer][3] single quotes for internal strings (read by the software and developers) and double quotes for stuff that could be shown to users.  Obviously not a critical issue, but I think it's good to have some sort of policy to keep things stable and consistent.

[1]: https://www.python.org/dev/peps/pep-0008/#string-quotes
[2]: http://programmers.stackexchange.com/questions/155176/single-quotes-vs-double-quotes
[3]: http://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python